microsoft-edge/devtools-guide-chromium/issues/index.md

```diff
@@ -20,118 +20,188 @@ keywords: microsoft edge, web development, f12 tools, devtools
+    :::image type="complex" source="../media/issues-tab-view-issue.msft.png" alt-text="None value in the SameSite column for the ck02 cookie in the Network tool" lightbox="../media/issues-tab-view-issue.msft.png":::
+       `None` value in the **SameSite** column for the `ck02` cookie in the **Network** tool
+    :::image-end:::
+
+1.  Scroll to view the item with a problem, such as the `ck02` cookie.  In the **SameSite** column, hover over the circled `i` \(![Cookie issue in Network tool](../media/network-tool-cookie-issue-icon.msft.png)\) next to **None**.  The resulting tooltip provides information similar to the **Issues** tool.
+
+    For the accessibility-testing demo page, issues have links to open the **Elements**, **Sources**, or **Network** tool.
+
+    :::image type="complex" source="../media/issues-tool-affected-resource-opens-elements-tool.msft.png" alt-text="Selecting an affected resource in the Issues tool for the accessibility demo page opens the Elements tool" lightbox="../media/issues-tool-affected-resource-opens-elements-tool.msft.png":::
+       Selecting an affected resource in the **Issues** tool for the accessibility demo page opens the **Elements** tool
+    :::image-end:::
+
+
+## Open issues from the DOM tree
+
+If an element has an issue, the DOM tree in the **Elements** tool shows wavy underlining under the element name.  You can either **Shift+click** the wavy-underlined element as described below, or right-click an element that has a wavy underline and then select **View issues**.
+
+To view an issue for an element that has wavy underlining in the DOM tree:
+
+1.  Open a page, such as the [accessibility-testing demo page](https://microsoftedge.github.io/DevToolsSamples/a11y-testing/page-with-errors.html), in a new tab or window.
+
+1.  Select `F12` to open DevTools, and then select the **Elements** tab.
+
+1.  In the DOM tree, expand `<body>` > `<header>` > `<form>`.  The `<input>` element has wavy underlining.
+
+    :::image type="complex" source="../media/issues-wavy-underlines-dom-tree.msft.png" alt-text="Wavy-underlined issues in the DOM tree in the Elements tool" lightbox="../media/issues-wavy-underlines-dom-tree.msft.png":::
+       Wavy-underlined issues in the **DOM tree** in the **Elements** tool
+    :::image-end:::
     
-    :::image type="complex" source="../media/issues-tab-view-request.msft.png" alt-text="View the affected cookie in the DevTools Network tool" lightbox="../media/issues-tab-view-request.msft.png":::
-       View the affected cookie in the DevTools **Network** tool  
-    :::image-end:::  
+1.  Hover over the `<input>` element.  A tooltip shows information about the issue.
 
-1.  Scroll to view the item with a problem:  for the following example, the `ck02` cookie.  Hover on the **SameSite** column to review the `None` value that the issue detected.  
+1.  Select `Shift` and then select the wavy-underlined element name.  The **Issues** tool opens and expands the issue about the element.
```

Comment:
I think we should adjust this wording to accommodate for keyboard-accessibile options. Options should be either "Shift + Click" (via mouse) or "open context menu" (which can be done via keyboard's context menu button or by using the keyboard to navigate to the ... menu next to node.)